packages/grafana-data/src/types/orgs.ts

```diff
@@ -0,0 +1,11 @@
+export interface UserOrgDTO {
+  orgId: number;
+  name: string;
+  role: Role;
+}
+
+export enum Role {
```

Comment:
This is already defined in https://github.com/grafana/grafana/blob/f58ec04c12d4678aaea04a9ab0325358e4d212b2/public/app/types/acl.ts#L1

Reply:
Thanks for pointing this out!

Reply:
Do you have any thoughts on whether the types defined in `grafana/public/app/types/acl.ts` should be moved to `@grafana/data`?

Reply:
some like OrgRole should I think, not sure about the rest

Reply:
@ryantxu  what's your thoughts on http api types, where should they be defined? app, grafana-data, or grafana-runtime . 

@kaydelaney can you rename it to OrgRole so it matches the one define in acl ?